netavark.spec.rpkg

```diff
@@ -45,8 +47,7 @@ OCI network stack.}
 %description %{_description}
 
 %prep
-#{{{ git_dir_setup_macro }}}
-%autosetup -Sgit -T -b 0 -n %{name}
+{{{ git_dir_setup_macro }}}
```

Comment:
FYI, `rpkg spec` in .packit.yaml expands this to `%setup foo bar baz` so that's why the last in .packit.sh checks for `%setup` and replaces with autosetup.